.github/container/test-maxtext.sh

```diff
@@ -187,7 +187,7 @@ export CUDA_DEVICE_MAX_CONNECTIONS=1
 
 export BASE_XLA_FLAGS=${BASE_XLA_FLAGS:---xla_gpu_enable_latency_hiding_scheduler=true 
                 --xla_gpu_enable_triton_gemm=false
-                --xla_gpu_simplify_all_fp_conversions 
+                --xla_allow_excess_precision
```

Comment:
This seems to be the default, why do we need to set it?

Reply:
agree

Reply:
Thanks, will remove it